overview.md

```diff
@@ -40,36 +38,36 @@ SubDAOs are expected to maintain a synergistic relationship with Maker Core thro
 
 Moving much of the developmental, operational, and governance work to SubDAOs makes Maker Core simpler and more resilient.
 
-### [Scopes and Scope Frameworks](maker-core/scopes-and-frameworks.md)
+### [Scopes and Scope Artifacts](maker-core/scopes-and-artifacts.md)
 
-Maker Core will consist of twelve Scopes. A Scope is a broad focus area for any work done to operate or improve the protocol. 
+Maker Core will consist of [five Scopes](maker-core/list-of-scopes.md). A Scope is a broad focus area for any work done to operate or improve the protocol. 
 
 The goal of introducing Scopes is to limit scope creep and ensure that any work done aligns with Maker Core's goals.
 
-Scope Frameworks define the rules that SubDAOs must adhere to. Scope Frameworks will be refined over time to maximize profit and minimize risk to Maker Core.
+Scope Artifacts define the rules that SubDAOs must adhere to. Scope Artifacts will be refined over time to maximize profit and minimize risk to Maker Core.
```

Comment:
I would add a sentence to introduce Scope Artifacts. 
Something like "Each Scope has a [Scope Artifact](maker-core/scopes-and-artifacts.md)." Followed by line #47